packages/api/portfolioDrafts/portfolio/createPortfolioStep.ts

```diff
@@ -70,13 +70,17 @@ export async function createPortfolioStepCommand(
       Key: {
         id: portfolioDraftId,
       },
-      UpdateExpression: "set #portfolioVariable = :portfolio, updated_at = :now",
+      UpdateExpression: `set #portfolioVariable = :portfolio, updated_at = :now,
+        #portfolioName = :portfolioName, num_portfolio_managers = :numOfManagers`,
       ExpressionAttributeNames: {
         "#portfolioVariable": PORTFOLIO_STEP,
+        "#portfolioName": "name",
```

Comment:
I thought the portfolioName was going to be set in the PORTFOLIO_STEP

Reply:
It was, but to align with the UI portfolio draft summary (see earlier comments) some of the data is being denormalized for easy access. Thanks for pointing the DELETE portfolioStep.